src/applications/facility-locator/tests/components/AppointmentInfo.unit.spec.jsx

```diff
@@ -2,86 +2,18 @@ import React from 'react';
 import { shallow } from 'enzyme';
 import { expect } from 'chai';
 import AppointmentInfo from '../../components/AppointmentInfo';
+import testFacilityV0 from '../../constants/mock-facility-v0.json';
+import testFacilityV1 from '../../constants/mock-facility-v1.json';
 
 describe('<AppointmentInfo>', () => {
-  const testFacility = {
-    id: 'vha_674BY',
-    type: 'facility',
-    attributes: {
-      access: {
-        health: [
-          {
-            service: 'Audiology',
-            new: null,
-            established: 0.5,
-          },
-          {
-            service: 'Cardiology',
-            new: null,
-            established: 0,
-          },
-          {
-            service: 'Dermatology',
-            new: 6.894736,
-            established: 8.363636,
-          },
-          {
-            service: 'Gastroenterology',
-            new: 3.5,
-            established: null,
-          },
-          {
-            service: 'Gynecology',
-            new: null,
-            established: 3.2,
-          },
-          {
-            service: 'MentalHealthCare',
-            new: 3.681818,
-            established: 3.716911,
-          },
-          {
-            service: 'Ophthalmology',
-            new: 3.5,
-            established: 4.175675,
-          },
-          {
-            service: 'Optometry',
-            new: 10.583333,
-            established: 3.75,
-          },
-          {
-            service: 'Orthopedics',
-            new: 7.428571,
-            established: 1.948717,
-          },
-          {
-            service: 'PrimaryCare',
-            new: 3.049382,
-            established: 1.279856,
-          },
-          {
-            service: 'SpecialtyCare',
-            new: 5.80916,
-            established: 4.679862,
-          },
-          {
-            service: 'WomensHealth',
-            new: null,
-            established: 3.2,
-          },
-        ],
-        effectiveDate: '2020-07-13',
-      },
-      feedback: {},
-      id: 'vha_674BY',
-      name: 'Austin VA Clinic',
-      facilityType: 'va_health_facility',
-    },
-  };
+  it('Should render appointment info component v0', () => {
+    const wrapper = shallow(<AppointmentInfo location={testFacilityV0} />);
+    expect(wrapper.type()).to.not.equal(null);
+    wrapper.unmount();
+  });
 
-  it('Should render access care component v1', () => {
-    const wrapper = shallow(<AppointmentInfo location={testFacility} />);
+  it('Should render appointment info  v1', () => {
+    const wrapper = shallow(<AppointmentInfo location={testFacilityV1} />);
     expect(wrapper.type()).to.not.equal(null);
```

Comment:
What do you think about adding a few more checks to this test?

This would be a good use case for snapshots IMO. But we could try [this module](https://www.npmjs.com/package/mocha-snapshots) to add snapshot tests for mocha. 

I [posted](https://dsva.slack.com/archives/CBU0KDSB1/p1595972612233200) in the platform Slack channel about this.

Reply:
more checks added